<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaleForm</class>
 <widget class="QWidget" name="ScaleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="ImageBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>30</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale Rate: 1.0</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="linearButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Linear</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="neighborButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Neighbor</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
